engine/pom.xml

```diff
@@ -588,9 +588,6 @@
 
     <profile>
       <id>h2-in-memory</id>
-      <activation>
-        <activeByDefault>true</activeByDefault>
-      </activation>
```

Comment:
@yanavasileva, does this impact any tests that do not need a specific DB to run (e.g. REST API tests)?

This would also mean that we (and any users that now build the project) will have to always specify a DB to run against. Is this the best approach?

Reply:
I checked this. Looks okay.